tap_azure_git/__init__.py

```diff
@@ -27,13 +27,13 @@
 REQUIRED_CONFIG_KEYS = ['start_date', 'user_name', 'access_token', 'org', 'repository']
 
 KEY_PROPERTIES = {
-    'commits': ['commitId'], # This is the SHA
+    'commits': ['id'],
     'pull_requests': ['artifactId'],
```

Comment:
It may look like pull_requests need to be updated, but this was already handled by the original author.

```
                # So pullRequestId isn't actually unique. There is a 'artifactId' parameter that is
                # unique, but, surprise surprise, the API doesn't actually include this property
                # when listing multiple PRs, so we need to construct it from the URL. Hilariously,
                # this ID also contains %2f for the later slashes instead of actual slashes.
                # Get the project_id and repo_id from the URL
                # TODO: not sure what type of exception to throw here if if the url isn't present
                # and matching this format.
                url_search = re.search('dev\\.azure\\.com/\w+/([-\w]+)/_apis/git/repositories/([-\w]+)', pr['url'])
                project_id = url_search.group(1)
                repo_id = url_search.group(2)
                pr['artifactId'] = "vstfs:///Git/PullRequestId/{}%2f{}%2f{}" \
                    .format(project_id, repo_id, pride)
```

Reply:
Yep, I vaguely remember dealing with this!